chainerx_cc/chainerx/context_test.cc

```diff
@@ -13,6 +13,17 @@
 #include "chainerx/native/native_device.h"
 #include "chainerx/testing/threading.h"
 
+#ifdef _WIN32
+int setenv(const char* name, const char* value, int overwrite) {
+    if (!overwrite) {
+        size_t requiredCount = 0;
+        auto err = getenv_s(&requiredCount, nullptr, 0, name);
+        if (err != 0 || requiredCount != 0) return err;
+    }
+    return _putenv_s(name, value);
+}
+#endif
```

Comment:
How about DRY-ing `{un}setenv`? Not entirely sure though there it should be defined.

Reply:
Also, could you follow the variable naming convention preferring snake case?